problemsets-mat223/modules/module1-exercises.tex

```diff
@@ -126,6 +126,52 @@
     		    \item Yes.
 		    \end{enumerate}
 		\end{solution}
+		
+		\prob
+		    Use set notations, vectors, or linear combinations of vectors to describe the following sets.
+		    \begin{enumerate}
+		        \item The set of vectors with rational components in $\R^2$.
```

Comment:
We use the word `coordinate` since `component` is associated with vector component. You need to spell it out a bit more. These are first years. E.g. "The set of vectors in R^2 whose coordinates are rational numbers."